src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -409,6 +412,11 @@ public virtual ClaimsIdentity CreateClaimsIdentity(SecurityToken securityToken,
         /// </summary>
         public CryptoProviderFactory CryptoProviderFactory { get; set; }
 
+        /// <summary>
+        /// Gets or sets a string that helps with setting breakpoints when debugging.
+        /// </summary>
+        public string DebugId { get; set; }
```

Comment:
Who is the audience of this property? is it for the library maintainers (us?), or also for the customer?
Is it used with Conditional breakpoints.

Reply:
mostly for developers, used for conditional breakpoints as I understand @brentschmaltz requested this.

Reply:
Right, its for us developers who maintain the library to set conditional breakpoints while debugging across products.